src/styles/todoapp.scss

```diff
@@ -46,6 +46,7 @@
 
     &.active {
       color: #737373;
+
```

Comment:
Do not forget about good formatting of the code (do not leave empty lines where it is not necessary).